Source/Scene/FXAA.js

```diff
@@ -50,6 +60,11 @@ define([
             owner : this
         });
         this._clearCommand = clearCommand;
+
+        this._qualityPreset = 39;
```

Comment:
I made them shader comments. I wasn't sure if we would expose them. I'm leaving `_qualityPreset` which is a preprocessor constant because we may expose it or change it for mobile. The possible values are 10-15, 20-29 or 39 (Its documented in a shader comment).